microfit/histogram/histogram_generator.py

```diff
@@ -1003,7 +1050,7 @@ def calculate_unisim_uncertainties(
             observation_dict[knob] = observations
             # If we get to this point without having either calculated a central value hist
             # or taken one from the cache, something is wrong
-            assert isinstance(central_value_hist, Histogram)
+            #assert isinstance(central_value_hist, Histogram) #TODO: This assert is failing even when central_value_hist is a histogram
```

Comment:
Are you sure this is not something that happens due to auto-reloading of classes in a Notebook? Because otherwise this function should have already always been failing, no?